### PR TITLE
chore(release): update changelog for 2021-08 release

### PR DESCRIPTION
**Pull request by devjgm**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7100)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,9 +52,11 @@
...
-## v1.30.0 - TBD
+## v1.31.0 - TBD
 
-## New GA Libraries
+## v1.30.0 - 2021-08
+
+### New GA Libraries
 * *BigQuery Storage* -- The [BigQuery Storage library](https://github.com/googleapis/google-cloud-cpp/tree/main/google/cloud/bigquery)
   is now GA.
 * *IAM* -- The [IAM libray](https://github.com/googleapis/google-cloud-cpp/tree/main/google/cloud/iam) is now GA.
```
Comment (dbolduc) [inline CHANGELOG.md:62]: s/libray/library/

File: CHANGELOG.md [hunk 2]
```diff
@@ -72,6 +74,12 @@
 
 [issue-2567]: https://github.com/googleapis/google-cloud-cpp/issues/2567
 
+**OTHER CHANGES**:
+* fix(bigtable): change default max mutations per batch from 100k to 1k (#7095)
+* cleanup(bigtable): strictly enforce bigtable limits in mutation batcher (#7036)
+* feat(bigtable): cap outstanding mutation number for mutation batcher (#7028)
+* feat(bigtable): start converting bigtable to use `g::c::Options` (#6949)
```
Comment (dbolduc) [inline CHANGELOG.md:81]: * The `MutationBatcher`'s default setting for max mutations per batch was reduced from 100k to 1k. The new value achieves better throughput and avoids errors from exhausting the server. (#7095)

File: CHANGELOG.md [hunk 3]
```diff
@@ -82,6 +90,26 @@
 
 [storage-mocking-link]: https://googleapis.dev/cpp/google-cloud-storage/latest/storage-mocking.html
 
+**OTHER CHANGES**:
+* feat(storage): discard handles after an error (#7088)
+* feat(storage): release download handles sooner (#7085)
+* feat(storage): support setting HTTP version (#7077)
+* fix(storage): cleanup interrupted downloads (#7064)
+* test(storage): make ScheduleForDelete() thread-safe (#7063)
```
Comment (coryan) [inline CHANGELOG.md:98]: test-only changes are probably not interesting to customers.

File: CHANGELOG.md [hunk 4]
```diff
@@ -82,6 +90,26 @@
...
+* fix(storage): avoid crashes after move (#7045)
+* feat(GCS+gRPC): upload checksums with last chunk (#7031)
+* fix(storage): use hash values in InsertObject() (#7025)
+* feat(storage): capture peer address for REST (#6994)
+* feat(GCS+gRPC): option to configure plugin (#6991)
+* feat(GCS+gRPC): checksums on ObjectInsert (#6967)
+* fix(GCS+gRPC): no hashes in Object resource (#6963)
+* feat(storage): use Base64 transforms (#6956)
```
Comment (coryan) [inline CHANGELOG.md:106]: Probably remove this one, it is basically a refactor, customers would not care.

**Review by coryan — APPROVED** [review]
A few nits, please fix and then LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (devjgm) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (devjgm) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (devjgm) [reply CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 4 above)
Comment (devjgm) [reply CHANGELOG.md]: Removed

**Pull request merged**